reference/tiflash/maintain.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Maintain a TiFlash Cluster
+summary: Learn common operations when you maintain a TiFlash cluster.
+category: reference
+---
+
+# Maintain a TiFlash Cluster
+
+This document describes common operations when you maintain a TiFlash cluster, including checking the version, node logout, troubleshooting, critical logs, and a system table.
+
+## Check the TiFlash version
+
+There are two ways to check the TiFlash version:
+
+- If the binary file name of TiFlash is `tiflash`, you can check the version by executing the `./tiflash version` command.
+
+    However, to execute the above command, you need to add the directory path which includes the `libtiflash_proxy.so` dynamic library to the `LD_LIBRARY_PATH` environment variable. This is because the running of TiFlash relies on the `libtiflash_proxy.so` dynamic library.
+
+    For example, when `tiflash` and `libtiflash_proxy.so` are in the same directory, you can first switch to this directory, and then use the following command to check the TiFlash version:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    LD_LIBRARY_PATH=./ ./tiflash version
+    ```
+
+- Check the TiFlash version by referring to the TiFlash log. For the log path, see [[logger] in the tiflash.toml configuration file](/reference/tiflash/configuration.md#configuration-file-tiflashtoml). For example:
+
+    ```
+    <information>: TiFlash version: TiFlash 0.2.0 master-375035282451103999f3863c691e2fc2
+    ```
+
+## Logout a TiFlash node
+
+Logouting a TiFlash node differs from [Scaling in the TiFlash node](/reference/tiflash/scale.md#scale-in-tiflash-node) in that the logout doesn't remove the node from TiDB Ansible; instead, it just safely shutdown the process.
```

Comment:
The reference anchor is not finalized because `scale.md` is not merged

Reply:
Please take a look at [L90](https://github.com/pingcap/docs/pull/2184/files#diff-23314ad704dd3050d84983065e6c7d7bR90). The anchor is decided: `#scale-in-a-tiflash-node`.